tester_app/client/webpack.client.base.config.js

```diff
@@ -0,0 +1,58 @@
+// Common client-side webpack configuration used by webpack.hot.config and webpack.rails.config.
+
+const webpack = require('webpack');
+const path = require('path');
+
+module.exports = {
+
+  // the project dir
+  context: __dirname,
+  entry: {
+
+    // See use of 'vendor' in the CommonsChunkPlugin inclusion below.
+    vendor: [
+      'babel-core/polyfill',
+      'jquery',
+      'jquery-ujs',
+      'react',
+      'react-dom',
+    ],
+
+    // This will contain the app entry points defined by webpack.hot.config and webpack.rails.config
+    app: [
+      './app/bundles/HelloWorld/startup/clientGlobals',
+    ],
+  },
+  resolve: {
+    extensions: ['', '.webpack.js', '.web.js', '.js', '.jsx', '.scss', '.css', 'config.js'],
+    alias: {
+      lib: path.join(process.cwd(), 'app', 'lib'),
```

Comment:
There were a lot of ../../'s when calling the loggerMiddleware. Adding this lets you call lib/middlewares/loggerMiddleware instead of diong tons of ../ stuff.

Here's the relevant line: https://github.com/shakacode/react_on_rails-generator-results-pre-0/blob/redux-server-rendering/tester_app/client/app/bundles/HelloWorld/store/helloWorldStore.jsx#L3.

@alexfedoseev does it for each bundle and the lib in FNG.